modules/mediawiki/manifests/php.pp

```diff
@@ -71,12 +71,10 @@
 
     php::extension { 'tideways-xhprof':
         ensure   => $profiling_ensure,
-        package_name => '',
```

Comment:
Why is this being removed?

Reply:
It didn't seem necessary. I can re-add it if it was.

Reply:
Removing it makes it default to php-${title} and so it tries to install a package that isn’t in the apt repo.

Reply:
Oh. OK, will re-add then. Thanks!